manager/manager/typeresolver.go

```diff
@@ -105,17 +105,19 @@ func (tr *typeResolver) ResolveTypes(config *common.Configuration, imports []*co
 			return nil, resolverError(config, fmt.Errorf("Failed to understand download url for %s: %v", r.Type, err))
 		}
 		if !existing[r.Type] {
-			f := &fetchUnit{}
-			for _, u := range urls {
-				if len(u) > 0 {
-					f.urls = append(f.urls, fetchableURL{urlRegistry, u})
-					// Add to existing map so it is not fetched multiple times.
-					existing[r.Type] = true
```

Comment:
This seems otherwise functionally the same to me except that this line could be hoisted up before the for loop? Or put another way, I don't see the need to introduce the if len(urls) > 0 { check.
Seems like a smaller and simpler change would be to just move this line above the for loop?
Is there a unit test you can add for this case? I've been testing with examples/package/cassandra.yaml, which worked for me, but would be better to have a unit test to prevent regressions.